common/djangoapps/student/views/dashboard.py

```diff
@@ -821,6 +822,7 @@ def student_dashboard(request):
         'nav_hidden': True,
         'inverted_programs': inverted_programs,
         'show_program_listing': ProgramsApiConfig.is_enabled(),
+        'show_journal_listing': journals_enabled(),
```

Comment:
**Plugin required:** The student dashboard needs to be pluggable.  The journals app would declare automatically that it has a new tab to display on the student dashboard.  There are at least 4 templates in this PR that needed to be manually updated and have hard-coded the conditional inclusion of journals.  With a pluggable dashboard, this wouldn't be needed.